app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -1,4 +1,4 @@
-child :content_facet => :content do
+child :content_facet => :content_facet_attributes do
```

Comment:
It's how rails works.  We're relying on update_attributes being called on
the host model during host update, and rails allows you to update and
create associated models via an association_attributes parameter.
On Apr 16, 2016 1:06 PM, "Eric D Helms" <notifications@github.com> wrote:

> In app/views/katello/api/v2/content_facet/show.json.rabl
> <https://github.com/Katello/katello/pull/5971#discussion_r59970472>:
>
> > @@ -1,4 +1,4 @@
> > -child :content_facet => :content do
> > +child :content_facet => :content_facet_attributes do
>
> If it's content_facet here why does it get mapped to have attributes
> attached at the end? You can point me at the code for this I just don't
> recall ever seeing it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5971/files/3f77419985a697694aeaf938848aeeff5cea9548#r59970472>
>